wiki/osu!_wiki/Contribution_guide/GitHub_web-based_editor/en.md

```diff
@@ -0,0 +1,70 @@
+# GitHub web-based editor
+
+*See also: [The github.dev web-based editor - GitHub Docs](https://docs.github.com/en/codespaces/the-githubdev-web-based-editor)*
+
+[github.dev](https://github.dev) is a publicly available, GitHub-tailored web version of [Visual Studio Code](https://code.visualstudio.com). Compared to GitHub's own web editor, it offers a lot more flexibility and control over the repository's contents. github.dev is a recommended way of editing the osu! wiki, especially when working on several articles at once.
+
+## Navigation
+
+*Note: To work with the osu! wiki, **[fork the `ppy/osu-wiki` repository](/wiki/osu!_wiki/Contribution_guide#editing-the-wiki).***
+
+To open osu! wiki in the editor, replace `github.com` in the URL of the repository with `github.dev`. Alternatively, open your fork on GitHub and press `.` (period).
+
+::: Infobox
+![](img/command-palette.gif "Example use of the command palette")
+:::
+
+While all menus are accessible from the editor's interface, the intended way of navigation is using the **command palette**:
+
+- Press `F1` and enter the name of a setting to open, or action to perform. If nothing shows up, explore the `≡` menu in the top left corner.
+- To open a file, press `Ctrl` + `P` and enter its name.
+
+## Branching
+
+After you have read [Best practices § Making changes](/wiki/osu!_wiki/Contribution_guide/Best_practices#making-edits), create a new branch to store your changes in.
+
+1. Click the current branch's name in the bottom left corner, or press `F1` and type `branch`.
+
+  - Select `Create new branch...`, enter the name of the branch, and press `Enter`.
+  - To help yourself remember what your work is roughly about, choose a descriptive name. As an example, for a German translation of [Beatmap Discussion](/wiki/Beatmap_Discussion), you could write `de-modding-v2`.
+
+2. To return to your changes, select the appropriate branch's name in the previous dropdown.
+
+## Editing
+
+### Existing files
+
+1. Press `Ctrl` + `P` and enter the name of a file you want to open, then press `Enter`. Loose matching is supported: for example, typing `nominators veto en` will bring up `wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto/en.md`.
+2. Edit the file as you wish. Uncommitted changes are **stored in your browser**, and you can return to them after leaving `github.dev`.
```

Comment:
```suggestion
2. Make your changes to the file. Uncommitted changes are **stored in your browser**, and you can return to them after leaving `github.dev`.
```

Reply:
I'd rather use less `changes` as today's Thanksgiving and the article is stuffed with it already